### PR TITLE
Feat/create envs file

### PR DESCRIPTION
**Pull request by ng11tk**
## Description

- Added the sync button for env synchronization in env list table in ui
- using multiple table for creating the envs in system 
- using json type column for env details in database
- resolves the issue #806 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist
- [x] Database schema is updated
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Looks good on large screens
- [x] Looks good on mobiles
- [x] Looks good on Tablets

Affected Apps
- [x] Store
    - [ ] SEO
    - [ ] Menu
    - [ ] Checkout
    - [ ] Add to Cart logic
    - [ ] Location Logic
    - [ ] Product page
    - [ ] Login
    - [x] Payment
    - [ ] Kiosk
- [x] Admin
    - [ ] Order
    - [ ] CRM
    - [ ] Product app
    - [ ] Analytics
    - [x] Settings
    - [ ] Brand
    - [ ] Config Builder
    - [ ] Coupons
    - [ ] Locations
    - [ ] Users
    - [ ] Inventory
- [x] Server
    - [x] Payment
    - [ ] Email
    - [ ] SMS
    - [ ] Notifications
    - [ ] Integration
    - [ ] Hasura auth
- [ ] Template Engine

### CODE REVIEW

File: admin/src/apps/settings/views/Listings/Envs/index.jsx [hunk 1]
```diff
@@ -52,10 +54,20 @@ const EnvsList = () => {
...
+            <Flex container alignItems="center" justifyContent='space-between' style={{ margin: '0px 0px 16px 16px' }}>
                 <Text as="h2" >
                     Envs({allSettings.length || 0})
                 </Text>
+                <ButtonGroup>
+                    <TextButton
+                        type="ghost"
+                        title='Synchronization after changes in env'
+                        size={24}
+                        onClick={() => axios.post("http://localhost:4000/server/api/envs")}
```
Comment (deepak-negi-web) [inline admin/src/apps/settings/views/Listings/Envs/index.jsx:66]: Don't hardcode the url, instead use the window.location.origin

File: server/entities/misc/controllers.js [hunk 2]
```diff
@@ -384,11 +399,110 @@ export const createEnvFiles = async () => {
    }
    return { server, store, admin }
 }
+export const createEnvFiles2 = async () => {
```
Comment (deepak-negi-web) [inline server/entities/misc/controllers.js:402]: if the createEnvFiles2 function is working then remove the createEnvFile and the ENVS, and use the proper naming

Comment (ng11tk) [reply server/entities/misc/controllers.js:402]: rishi asked me to name the new function as createEnvaFile2 and if we are using this function as 2 than i thought maybe we should keep first function

File: admin/src/apps/settings/views/Listings/Envs/index.jsx (hunk 1 above)
Comment (ng11tk) [reply admin/src/apps/settings/views/Listings/Envs/index.jsx]: done

**Review by ng11tk — COMMENTED** [review]
changed as requested

File: server/entities/misc/controllers.js (hunk 2 above)
Comment (ng11tk) [reply server/entities/misc/controllers.js:402]: @futuregridtechnologies what you say about it

**Review by shubhamz950 — APPROVED** [review]

**Pull request merged**